Makefile

```diff
@@ -64,6 +64,11 @@ smithy-build-%: gen-repo-mod-replace
 smithy-clean:
 	cd codegen && ./gradlew clean
 
+smithy-go-publish-local:
+	rm -rf /tmp/smithy-go-local
+	git clone https://github.com/aws/smithy-go /tmp/smithy-go-local
+	make -C /tmp/smithy-go-local smithy-publish-local
```

Comment:
should we do `smithy-clean` before `publish-local` ?

Reply:
Shouldn't be needed since the `smithy-go` repo is freshly created before the make command is run. But it shouldn't hurt if there is some caching that gits stuck out there.